main/src/library/Concurrent/Channel/Ticker.flix

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2018 Magnus Madsen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Concurrent/Channel {
+
```

Comment:
This could also be one declaration

Reply:
As in `namespace Concurrent/Ticker` if I remember correctly :)

Reply:
But still under Channel right? so `Concurrent/Channel/Ticker`